releases/3.0.10.md

```diff
@@ -1,16 +1,20 @@
 ---
-title: TiDB 3.0.10 Release Notes
+title: TiDB 3.0.10 Release Notes (deprecated)
 category: Releases
 ---
 
-# TiDB 3.0.10 Release Notes
+# TiDB 3.0.10 Release Notes (deprecated)
 
 Release date: February 20, 2020
 
 TiDB version: 3.0.10
 
 TiDB Ansible version: 3.0.10
 
+> **Warning:**
+>
+> This version is deprecated.
```

Comment:
@scsldb 

- How about suggesting users choose a specific version? So users know which version to use.
- Please help confirm whether it's appropriate to say `deprecated` here. Users might be confused. Would it be better to give some explanations?